docs/source/developers.rst

```diff
@@ -0,0 +1,12 @@
+##########
+Developers
+##########
+
+This is a list of the developers of Tin, from
+latest to oldest. Who knows, if you contribute enough,
+you might make your way up here!
+
+* \(2027) Aarush Deshpande
+* \(2024) Krishnan Shankar
+* \(2020) John Beutner
```

Comment:
@Laur04 would you like a spot here?

Reply:
Oops didn't see her on the banner at the bottom ;)